commandlineparser/src/Parser.cpp

```diff
@@ -72,19 +70,24 @@ std::vector<std::string> cmdParser::Parser::help_qualifier_keys_finder()
 	return help_keys;
 }
 
-void cmdParser::Parser::default_help(std::vector<std::string>& keys)const
+void cmdParser::Parser::print(const std::vector<std::string>&keys, std::function<void(const std::string target_key)> print_help, std::string title)const
 {
 	std::cout << "\n **************WELCOME TO OPTIONS OF THE LIBRARY *********************" << std::endl;
 	std::cout << "[FORMAT]" << std::endl;
 	std::cout << "-------------------------------------------------------------------------------------" << std::endl;
-	std::cout << "Short_Command" << "\t" << "Long_Command" << "\t" << "Short_Description" << "\t\t" << "Long_Description" << std::endl;
+	std::cout << "Short_Command" << "\t" << "Long_Command" << "\t" << title << std::endl;
 	std::cout << "-------------------------------------------------------------------------------------" << std::endl;
-	
-	auto print = [&](const std::string target_key) 
-	{     
+	for_each(keys.begin(), keys.end(), print_help);
+
+}
+
+void cmdParser::Parser::default_help(const std::vector<std::string>& keys)const
+{
+	auto print_default_help = [&](const std::string target_key)
+	{
 		auto itr = command_list.at(target_key);
-		std::cout<<"\n"<<itr->get_option_short_command()<<"\t\t"<<itr->get_option_long_command()<<"\t\t"<< itr->get_option_short_description()<<"\t\t"<<itr->get_option_long_description()<<std::endl;
+		std::cout << "\n" << itr->get_option_short_command() << "\t\t" << itr->get_option_long_command() << "\t\t" << itr->get_option_short_description() << "\t\t" << itr->get_option_long_description() << std::endl;
```

Comment:
use either "\n" or std::endl... you will not require "\n" if  you are using std::endl after every std::cout